subsys/testsuite/include/tc_util.h

```diff
@@ -15,7 +15,13 @@
 #include <shell/shell.h>
 #include <sys/printk.h>
 
+#if defined CONFIG_ZTEST_TC_UTIL_USER_OVERRIDE
+#include <tc_util_user_override.h>
```

Comment:
Is this going to be included per test wishing to override? Can you please add a test that uses this new functionalty? We also need documentation for this in doc/guides/test/

Reply:
I added a test and some documentation in the latest commit.